src/Raven.Studio/typescript/components/pages/database/tasks/ongoingTasks/editTasks/editGenAiTask/partials/fields/EditGenAiTaskBasicFields.tsx

```diff
@@ -0,0 +1,123 @@
+import ButtonWithSpinner from "components/common/ButtonWithSpinner";
+import { FormInput, FormSwitch, FormGroup, FormLabel, FormSelect } from "components/common/Form";
+import RichAlert from "components/common/RichAlert";
+import { SelectOption } from "components/common/select/Select";
+import { databaseSelectors } from "components/common/shell/databaseSliceSelectors";
+import useBoolean from "components/hooks/useBoolean";
+import EditConnectionStrings from "components/pages/database/settings/connectionStrings/EditConnectionStrings";
+import { useAppDispatch, useAppSelector } from "components/store";
+import { sortBy } from "lodash";
+import { useAsync } from "react-async-hook";
+import { editGenAiTaskActions, editGenAiTaskSelectors } from "../../store/editGenAiTaskSlice";
+import EditGenAiTaskNodeField from "./EditGenAiTaskNodeField";
+import InputGroup from "react-bootstrap/InputGroup";
+import { useServices } from "components/hooks/useServices";
+import { useFormContext } from "react-hook-form";
+import { EditGenAiTaskFormData } from "../../utils/editGenAiTaskValidation";
+
+type OngoingTaskState = Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskState;
+
+export default function EditGenAiTaskBasicFields() {
+    const dispatch = useAppDispatch();
+
+    const isNewTask = useAppSelector(editGenAiTaskSelectors.isNewTask);
+    const isEncrypted = useAppSelector(databaseSelectors.activeDatabase)?.isEncrypted ?? false;
```

Comment:
`?? false` isnt strictly necessary undefined is already falsy
```suggestion
    const isEncrypted = useAppSelector(databaseSelectors.activeDatabase)?.isEncrypted;
```